kas/config/mender-raspberrypi.yaml

```diff
@@ -12,7 +12,7 @@ local_conf_header:
     IMAGE_FSTYPES:remove = " rpi-sdimg wic.bz2 wic.bmap"
     SDIMG_ROOTFS_TYPE = "ext4"
 
-    MENDER_STORAGE_TOTAL_SIZE_MB ?= "2048"
+    MENDER_STORAGE_TOTAL_SIZE_MB ?= "2560"
```

Comment:
What's the reason for this change? Did you get warnings about too little rootfs space when building for scarthgap?

Reply:
Ok, we'll have to review this new value and possibly backport it to kirkstone so that users can upgrade from kirkstone to scarthgap etc...the change in size might cause some problem during the upgrade as I don't think it supports resizing the root partition.

Reply:
I've tested it and changing this value does break OTA updates, however it is better to break it now and increase it to 4096MB to give us a buffer for future changes. The change to 4096MB would provide a buffer of about 1.6GB for the root-fs (per A/B partition).

I've backported this setting to the kirkstone branch, and will rebase shortly (after rebasing the current scarthgap branch)